.buildkite/build-static-release.sh

```diff
@@ -28,7 +32,21 @@ esac
 
 echo will run with $CONFIG_OPTS
 
-./bazel build //main:sorbet --strip=always $CONFIG_OPTS
+case "$platform" in
+  darwin-x86_64|darwin-arm64)
+    ./bazel build //main:sorbet --strip=always $CONFIG_OPTS
+    mv bazel-bin/main/sorbet sorbet_x86_64
+
+    ./bazel build //main:sorbet --strip=always $CONFIG_OPTS --config=release-mac-arm64-cross
+    mv bazel-bin/main/sorbet sorbet_arm64
+
+    lipo -create -output bazel-bin/main/sorbet sorbet_x86_64 sorbet_arm64
+    rm sorbet_x86_64 sorbet_arm64
```

Comment:
I believe that this command is hanging, because `sorbet_x86_64` and `sorbet_arm64` are copied out of the bazel sandbox, and all files in the bazel sandbox are write protected (`r-xr-xr-x` / `555`).

I think that you have to `-f` this
```suggestion
    rm -f sorbet_x86_64 sorbet_arm64
```

Reply:
I would think that would just error, rather than hang?